oidc-client/base/virtual-service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+  name: oauth-route
+  namespace: kubeflow
+spec:
+  gateways:
+  - kubeflow/kubeflow-gateway
```

Comment:
virtual service should be deployed in the same namespace as kubeflow, so the ns prefix (kubeflow/) is not required